### ISSUE TITLE
Require a specific columns in a csv file 

### ISSUE THREAD

**Issue by santhoshnumberone** [heartexlabs/label-studio#2292]
I am trying to replicate this tutorial [Building your own object detector — PyTorch vs TensorFlow and how to even get started?](https://towardsdatascience.com/building-your-own-object-detector-pytorch-vs-tensorflow-and-how-to-even-get-started-1d314691d4ae)

I need the csv data in a specific format like shown [here](https://towardsdatascience.com/building-your-own-object-detector-pytorch-vs-tensorflow-and-how-to-even-get-started-1d314691d4ae?#fdfa)
with column names`label`,`xmin`,`ymin`,`xmax`,`ymax`,`filename`,`width`,`height`

I don't seem to find this kind of csv data I can download after annotation

Am I missing something here? or is it not possible to get the data this specific format currently?
